CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing Guidelines for firecamp.io
+
+Thank you for considering contributing to firecamp.io! To ensure smooth collaboration and maintain a consistent workflow, please follow the guidelines below.
+
+## Commit Guidelines
+
+- We encourage [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) format for your commit messages.
+- Keep your commit messages concise, clear, and descriptive.
+- Make sure to include a brief summary of the changes made in the commit.
+
+```
+The commit message should be structured as follows:
+
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+
+```
+Example Commit Message:
+
+```
+feat(auth): Add new authentication module
+
+```
+
+## Fork and Pull Request Workflow
+
+1. Fork the firecamp.io repository to your GitHub account.
+2. Create a new branch for your changes. Use a descriptive and meaningful name for the branch that reflects the purpose of your contribution.
+3. Make your changes in the branch, following the project's coding style and guidelines.
+4. Commit your changes with appropriate messages, adhering to the commit guidelines mentioned above.
+5. Push your branch to your forked repository.
+6. Create a pull request (PR) from your branch to the main firecamp.io repository.
+7. Provide a clear and concise title for your pull request, summarizing the changes made.
+8. In the PR description, provide a detailed explanation of the problem you solved or the feature you added. If applicable, include relevant screenshots or a video showcasing the changes.
+9. Tag your PR with one or more relevant labels from the following options: `graphql`, `ui`, `documentation`, `rest`, `backend`, or any other appropriate labels.
+
+Example branch name
+```
+auth-module
+```
+
+Example PR title
+```
+feat(auth): Add new authentication module
+
+```
+
+Example PR description
+
+```
+## Description
+This pull request adds a new authentication module to improve user security and access control.
+
+## Changes Made
+- Implemented JWT-based authentication
+- Added user registration and login endpoints
+- Enhanced password encryption and storage
+
+## Screenshots
+![Screenshot 1](/images/screenshot1.png)
+![Screenshot 2](/images/screenshot2.png)
+
+Fixes #123
+
```

Comment:
Is this section required after merging #14? Or we can reference this template. I'm not sure but happy to have your thoughts. 

It's not a big issue, we can keep improving the contributing.md.  @Deepakanandrao

Reply:
No, you are right. We can remove the examples and keep the general guidelines from both the sections